ganttproject/src/biz/ganttproject/impex/csv/TaskRecords.java

```diff
@@ -166,6 +169,14 @@ protected boolean doProcess(SpreadsheetRecord record) {
         builder = builder.withCompletion(Integer.parseInt(completion));
       }
     }
+    if (record.isSet(TaskDefaultColumn.COLOR.getName())) {
+      try {
+        Color taskColor = ColorConvertion.determineColor(getOrNull(record, TaskFields.COLOR.toString()));
+        builder.withColor(taskColor);
+      } catch (AssertionError e) {
```

Comment:
I assume this handles the case when color is not in hexadecimal format. 
Catching `AssertionError` is kinda unusual thing, though. I suggest creating a method `boolean ColorOption.Util.isValidColor(String)` which would return `true` if argument matches the regular expression and check the string before converting to color.